Embedded/MaxMix/Display.ino

```diff
@@ -35,23 +35,41 @@ void DisplayAppNavigateScreen(Adafruit_SSD1306* display, Item* item, int8_t item
 {
   display->clearDisplay();
 
-  // Left Arrow
-  if(itemIndex > 0)
-  {
+
+  if(settings.continuousScroll && itemCount > 0)
```

Comment:
We are trying to not use global variables in functions outside of the Maxmix.ino file.  
Could you add a new argument to this function "uint8_t continuousScroll" and remove the reference to settings here?

Also if there is only 1 item, the arrows should be hidden. Therefore you may want to have 2 if statements like so:
`if(continuousScroll)
    if(itemCount > 1)
    ...`

Reply:
Alright. Do I squash these with the initial commits or do I leave them a separate commit?

Reply:
Yeah leve it separate, it’s ok. 
Thanks a lot for the contribution, it’s great!